packages/pg-v5/lib/host.js

```diff
@@ -6,7 +6,7 @@ module.exports = function (addon) {
   let host = process.env.HEROKU_POSTGRESQL_HOST
   let starterHost = process.env.HEROKU_POSTGRESQL_STARTER_HOST
```

Comment:
Should we change this to `HEROKU_POSTGRESQL_ESSENTIAL_HOST` ?

Reply:
Oh, yeah. Good catch.